app/src/main/res/xml/preference_payment_setting.xml

```diff
@@ -72,6 +72,10 @@
             app:key="@string/payment_preference_mobile_banking_kbank_key"
             app:title="@string/payment_preference_mobile_banking_kbank_title" />
 
+        <CheckBoxPreference
+            app:defaultValue="true"
+            app:key="@string/payment_preference_mobile_banking_ktb_key"
+            app:title="@string/payment_preference_mobile_banking_ktb_title" />
```

Comment:
Should we put an empty line at the bottom?

Reply:
https://github.com/omise/omise-android/pull/248/commits/2bf34b7d983ecb0526b7d62fc99e5048388a8144